test_spectral_norm.py

```diff
@@ -33,16 +33,17 @@ def permute_shape(shape: Tuple, order: Tuple) -> Tuple:
     """Take a shape and permute it. i.e. change the order of the elements
 
     Args:
-        shape: a tuple
-        order: a tuple with values [0, len(shape)), an ordering of `shape`
+        shape: Input shape
+        order: Ordering of shape (i.e. [0, len(shape)))
```

Comment:
can we revert this?

Reply:
Well, if you insist. I think that it is trivial to specify the types again in the docstring, since we are using type-hinting.

Reply:
i understand.  though i guess just make it clear that the order is not equal to [0, n) but rather takes on all of those values.